src/Paraglider/Rx.purs

```diff
@@ -222,3 +225,21 @@ toClosure upstreamEffect closure = makeEvent \downstreamPush -> do
   upstream <- upstreamEffect
   downstreamPush (closure upstream)
   pure (pure unit)
+
+-- interop
+
+data Observable :: forall k. k -> Type
```

Comment:
Now the `Rx.purs` file can actually be only for this

Reply:
Done!